documentation/project-guides/presentation_principles.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: How to Present Data Science Findings to Clients -- 3 Principles and a Case Study
+author: Handi Xie
+parent: Data Science Project Documentation Guidelines
+nav_order: 7
+---
+
+# How to Present Data Science Findings to Clients -- 3 Principles and a Case Study
+
+# Background
+
+Students enrolled in Spark! courses/practicum are usually loaded with adequate technical skills to perform data analytics. However, there might not be a consensus how to present our findings/results to a client. Below is a blog post, originated from a conversation with a FA23 Spark! team, that could give you some insights on: **How to Present Data Science Findings to Clients**.
+
+# 3 Principles
+1. Use data-driven strategies for storytellings. Support your conclusions and hypothesis with data.
+2. Know your audience, level your presentations with your clients and explain in ways they can understand.
+3. Embrace [Ownership Principle](https://www.amazon.jobs/content/en/our-workplace/leadership-principles). Task shouldn't be limited to completing the code/presentation/report; avoid the mindset that the rest is "not my job". Consider the bigger picture of how findings could benefit the client.
+
+# Case Study
+## Case Study Background
+
+The case study used in the blog post is from a Spark! data science project that centers around unanswered survey data, where we want to help analyze survey data over the past 8-9 years. For a desensitized data, please refer to [this spreadsheet](https://docs.google.com/spreadsheets/d/1wBBV9612tsHR8MsMihHI3gY7ebE2YMsE/edit?usp=sharing&ouid=101499547981540031808&rtpof=true&sd=true) (Every data entry is populated with augmented data).
+
+## Case Study Practice
+
+Before going deeper into the blog post, imagine you were given a tabular survey data for 20 survey responders across 10 years e.g. "Effectiveness of head coach's communication skills (on a scale from 1 to 4)". How would you use it to answer the following questions, "How have athletes' perceptions of coach communication evolved over the 8-9 year period?". Se the highlighted two rows below.
+
+> Survey Data

```

Comment:
Also remove the screenshots of the survey data and head coach change

Reply:
Hi Michelle, I have made changes according to your comments. Thank you for that!